docs/rules/order-properties.md

```diff
@@ -31,19 +31,28 @@ Examples of **correct** code for this rule:
 
 ### Options
 
-Pass an array of top-level package properties to lint sorting on only those
-properties. All properties not in this collection will be ignored.
+```js
+"package-json/order-properties": ["error", {
+    order: "sort-package-json"
+}]
+```
 
-Example:
+#### Order
 
-```js
-"package-json/order-properties": ["error", [
-    "name",
-    "version" // Ensure only that name precedes version
-]]
+The `order` property specifies the sorting order of package properties. Pass in:
+
+-   "legacy" - to order properties specified by [npm documentation](https://docs.npmjs.com/cli/v10/configuring-npm/package-json).
+-   "sort-package-json" - to order properties by the default order specified in [sort-package-json](https://github.com/keithamus/sort-package-json).
+-   Array<string> - to specify an array of top-level package properties to lint sorting on only those
```

Comment:
[Docs] The last item shows up as _"Array - to specify ..."_ in GitHub's rendered pages. Best to backtick.

```suggestion
-   `"legacy"` - to order properties specified by [npm documentation](https://docs.npmjs.com/cli/v10/configuring-npm/package-json).
-   `"sort-package-json"` - to order properties by the default order specified in [sort-package-json](https://github.com/keithamus/sort-package-json).
-   `Array<string>` - to specify an array of top-level package properties to lint sorting on only those
```